src/util/iroha/commands.js

```diff
@@ -194,12 +195,33 @@ const transferAssetWithFee = (privateKeys, quorum, {
   feeType,
   timeoutLimit = DEFAULT_TIMEOUT_LIMIT
 }) => {
-  let feeAccountId = `${feeType}@d3`
-  let txClient = newCommandService()
-  let senderTx = txHelper.addCommand(txHelper.emptyTransaction(), 'transferAsset', { srcAccountId, destAccountId, assetId, description, amount })
-  if (fee > 0) {
-    senderTx = txHelper.addCommand(senderTx, 'transferAsset', { srcAccountId, destAccountId: feeAccountId, assetId, description, amount: fee })
+  const txClient = newCommandService()
+
+  let senderTx = txHelper.addCommand(
+    txHelper.emptyTransaction(),
+    'transferAsset',
+    { srcAccountId, destAccountId, assetId, description, amount }
+  )
+
+  if (feeType === FeeTypes.WITHDRAWAL) {
+    if (fee > 0) {
+      senderTx = txHelper.addCommand(
+        senderTx,
+        'transferAsset',
+        { srcAccountId, destAccountId, assetId, description: 'withdrawal fee', amount: fee }
+      )
+    }
+  } else {
+    if (fee > 0) {
```

Comment:
I think it's small code duplication and we can move this two `if (fee > 0)` outside `if (feeType === FeeTypes.WITHDRAWAL)` block.